HelpSource/Tutorials/Getting-Started/02-First-Steps.schelp

```diff
@@ -96,45 +96,43 @@ line 3 char 10:
 
 Usually the problem actually occurs a little before that, so that's where you should look.  In this case, it's the lack of a semi-colon at the end of the previous line.  Note that each line of code ends normally with a semi-colon.  This is how you separate lines of code in SC. Since we didn't have a semi-colon between the two lines we have gotten an error. 
 
-Using semi-colons it's possible to have more than one line of code in the same line of text. This can be handy for execution.
+Using semi-colons, it's possible to have more than one code statement on the same line of text. This can be handy if you don't want to have to select multiple lines to execute related code statements.
```

Comment:
imo this section should actually just be deleted; multiple statements on a single line is bad style and we shouldn't encourage it in our primary tutorial

Reply:
haha, I agree, will do. I felt like my edit was a little contrived, but I was trying to think about it from the point of view of someone who genuinely believed it to be a benefit